accession-commons-core/src/main/java/uk/ac/ebi/ampt2d/commons/accession/core/models/AccessionWrapper.java

```diff
@@ -18,12 +18,12 @@
 package uk.ac.ebi.ampt2d.commons.accession.core.models;
 
 /**
- * Wrapper containing the object that has been accessioned, as well as additional information like the accession or a
- * flag indicating whether the accession is active. To be used in the service layer.
+ * Wrapper containing the object that has been accessioned, as well as additional information indicating whether
+ * the accession is active. To be used in the service layer.
```

Comment:
There is no information in this class "indicating whethere the accession is active".